bundle/src/main/java/com/adobe/acs/commons/throttling/RequestThrottler.java

```diff
@@ -225,6 +225,14 @@ protected void delay(long ms) {
         try {
             Thread.sleep(ms);
         } catch (InterruptedException e) {
+            // We no longer perform thread interrupts, so something else in AEM is causing this interruption.
```

Comment:
@badvision @kwin 
this is the best way to handle this right - since w should either be restoring the interrupted status or throwing the exception (i really dont want to get into changing signatures w/ this fix)

Reply:
and sorry - that was a question, not a statement :P  ... thanks kwin!